beetsplug/fetchart.py

```diff
@@ -447,6 +502,13 @@ def __init__(self):
         if not self.config['google_key'].get() and \
                 u'google' in available_sources:
             available_sources.remove(u'google')
+        if not self.config['fanarttv_key'].get() and \
+                u'fanarttv' in available_sources:
```

Comment:
@wordofglass I just noticed that the "you might want a fanart.tv personal key" message prints unconditionally when the `fetchart` plugin is enabled (even if the `fanart` source is not enabled). I think that condition needs to check `self.config['sources']`, not `available_sources`. Could you please fix this?